stdcommands/forex/forex.go

```diff
@@ -41,7 +42,7 @@ var Command = &commands.YAGCommand{
 		if (to == nil) || (from == nil) {
 			return "Invalid currency code.\nCheck out available codes on: <https://api.exchangerate.host/symbols>", nil
 		}
-		output, err := requestAPI("https://api.exchangerate.host/convert?from=" + from.Code + "&to=" + to.Code + "&amount=" + amount.Str())
+		output, err := requestAPI("https://api.exchangerate.host/convert?from=" + from.Code + "&to=" + to.Code + "&amount=" + strconv.FormatFloat(amount, 'f', -1, 64))
```

Comment:
This may be better written as a call to Sprintf.

Reply:
Sounds good, thanks